onnx/onnx.proto

```diff
@@ -68,12 +68,16 @@ message AttributeProto {
   optional bytes s = 4;               // UTF-8 string
   optional TensorProto t = 5;         // tensor value
   optional GraphProto g = 6;          // graph
-
+  optional uint64 ui = 12;            // uint
```

Comment:
do we have concrete use cases for unsigned and high-precision numbers as attributes? I'm not opposed to them, but just curious whether we really need them at this point (few more parts for a switch and also less natural as now float-valued ones need to be looked up in both `f` and `d`).